### DISCUSSION TITLE
Update on the Release 6.0.0 - Angular upgrade

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-coKreat/Community#67] [Announcements]
Sunbird platform is planned to upgrade the front-end applications of Angular to the community supported version of Angular. 

Current latest version of Angular is 15. Angular community always supports 3 previous releases of Angular. So as sunbird platform front-end applications which are using Angular, we want to be in the supported versions of Angular. 

Please refer the below **discussion thread of SunbirdEd for Angular upgrade.**
https://github.com/orgs/Sunbird-Ed/discussions/403

As part of this, CoKreat team also upgrading the angular to the same supported version of SunibrdED(to make consistent & adopter friendly to contribute if all are in same Angular version). 

We are planned to upgrade to Angular 14. 

Please find the below branches of Angular upgrade
**Angular 12** - https://github.com/Sunbird-coKreat/creation-portal/tree/6.0.0-ng-12
**Angular 13** - TBU(InProgress)
**Angular 14** - TBU(InProgress)

**Note:** 
The development/PRs for Release 6.0.0.0 needs to be on Angular version 14 otherwise merging them will not be possible. This is to keep you informed so that the upcoming work is planned accordingly. 

@vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @kiranharidas187
@sanyapawah @aayushkedia95 @ChakshuGautam @abhay-samagra
@pkranga @mathewjpallan @mahesh-maney
@pritamps @karn-nahata
@coolbung @parthlawate
@kameshbhr @mmanimegalai @anandvarada
@SanthoshVasabhaktula @vrayulu
@ashokreddy1208 @gandham-santhosh @rajeevsathish @vinukumar-vs @Krishnaj20 @swayangjit
@pallakartheekreddy @rhwarrier @anandp504 @reshmi-nair @surendrasinghs @mmanimegalai @maheshkumargangula
@madhucr @alok-os

**Comment by abhay-samagra** [comment]
Hi @vinukumar-vs - We have raised two PRs,
https://github.com/Sunbird-coKreat/program-service/pull/304
https://github.com/Sunbird-coKreat/program-service/pull/303

Plz have a look.

Also, as per my understanding, they will be a part of 6.0.0, right?
